core/kazoo_fixturedb/doc/README.md

```diff
@@ -136,50 +151,58 @@ doc_id, attachment_name, attachment_file_name
 201710-vm_message0000000000000000000001, vm-new_message.wav, 5cef38cd0260632cf119c6fcf0c44f58.att
 ```
 
-When you're adding an attachment use `kz_fixturedb_util:get_att_path/3,4` to get the file path and save it there then use `kz_fixturedb_util:add_att_path_to_index/3,4` to add the attachment file name to the `attachment-index.csv` file.
-
-Putting an attachment is just returning the updated document JObj back. Deleting an attachment is returning tuple `{'ok', JObj}` whic JObj just have `id` and `rev`. Both put and delete will return error tuple if the JSON document file does not exists in database.
+Putting an attachment is just returning the updated document JObj back. Deleting an attachment is returning tuple `{'ok', JObj}` which the JObj just have the `id` and `rev`. Both put and delete will return error tuple if the JSON document file does not exists in database.
```

Comment:
Delete should include `"_deleted":true` too I believe. Not sure if code checks for this but to avoid surprises later.

Reply:
nope: http://docs.couchdb.org/en/2.1.1/api/document/attachments.html#delete--db-docid-attname

Reply:
TIL